Ch09/Ch09.md

```diff
@@ -805,19 +801,19 @@ This is more flexible, but it has a downside. If you're querying over a large da
 
 You aren't limited to sorting by the fields on the document. You also have the following sorting options:
 
-* `order by score()`&mdash;order the results by how closely they match the `where` clause (useful if you are using full text search or using `OR` in 
+* `order by score()` &mdash; order the results by how closely they match the `where` clause (useful if you are using full text search or using `OR` in 
   the `where` clause).
-* `order by random()`&mdash;random order, useful for selecting a random result from the query.
-* `order by spatialdistance()`&mdash;useful for spatial queries and will be discussed later in this chapter.
-* `order by count()`/`order by sum()`&mdash;allows ordering the results of queries using `group by`, discussed later in this chapter. 
+* `order by random()` &mdash; random order, useful for selecting a random result from the query.
+* `order by spatial.distance()` &mdash; useful for spatial queries which will be discussed later in this chapter.
+* `order by count()` / `order by sum()` &mdash; allows ordering the results of queries using `group by`, discussed later in this chapter. 
 
 The sort order can also be impacted by how you want RavenDB to sort the fields. For example, consider the following query:
 `from Products order by PricePerUnit desc`. If you run it, you'll get some peculiar results, as shown in Figure 9.13.
 
 ![Unexpected ordering with numeric order by](./Ch09/img13.png)
 
 The issue is that RavenDB doesn't know the type of the `PricePerUnit` field, and as such, it defaults to lexical ordering. In this case, this means that
-`7` is sorted after `62.5`. In our `order by` clause, we need to let RavenDB know what kind of sorting we want to do. An example of how to do just that can be seen in Listing 9.21.
+`7` is sorted before `62.5`. In our `order by` clause, we need to let RavenDB know what kind of sorting we want to do. An example of how to do just that can be seen in Listing 9.21.
```

Comment:
Pretty sure that this is correct. We are comparing lexically, so 7 is greater than 62, no?

Reply:
Fixed.

=> The issue is that RavenDB doesn’t know the type of the PricePerUnit field, and therefore it defaults to lexical ordering, which means that 7 will show before 62.5. 
In order to get the actual descending results from our `order by` clause, we need to let RavenDB know what kind of sorting we want to do. 
An example of how to do just that can be seen in Listing 9.21.